.github/workflows/java-cd.yml

```diff
@@ -211,6 +211,29 @@ jobs:
                   echo "Deployment ${{ env.DEPLOYMENT_ID }} was unsuccessful with status $DEPLOYMENT_STATUS"
                   exit 1
 
+    test-deployment-on-all-architectures:
+        needs: [set-release-version, load-platform-matrix, publish-to-maven-central-deployment]
+        env:
+            JAVA_VERSION: "11"
+            RELEASE_VERSION: ${{ needs.set-release-version.outputs.RELEASE_VERSION }}
+        strategy:
+            # Run all jobs
+            fail-fast: false
+            matrix:
+                host: ${{ fromJson(needs.load-platform-matrix.outputs.PLATFORM_MATRIX) }}
+
+        runs-on: ${{ matrix.host.RUNNER }}
+        steps:
+            - name: Start Redis server
```

Comment:
Why redis, not valkey? Why you don't reuse `install-valkey` action?

Reply:
i think it was because the procedure to install redis is easier than valkey. i can try out install-valkey

Reply:
Is it really that different? Valkey is just a fork of Redis.

Reply:
actually, valkey doesn't have an installer package and you need to install from source. 
but the install-valkey workflow is available to do that for us.